inc/deploygroup.class.php

```diff
@@ -789,6 +789,7 @@ public function showForComputer($computers_id)
         foreach ($iterator as $data) {
             $this->getFromDB($data['plugin_glpiinventory_deploygroups_id']);
             echo "<tr>";
+            echo "<td></td>";
```

Comment:
To ensure there is the same column count as for static data.

```suggestion
            if ($canedit) {
                echo "<td></td>";
            }
```